odl/contrib/shearlab/test/operator_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2014-2018 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Unit tests for the ODL-shearlab integration."""
+
+import pytest
+import numpy as np
+import odl
+import odl.contrib.shearlab
+from odl.util.testutils import all_almost_equal, simple_fixture
+
+
+dtype = simple_fixture('dtype', ['float32', 'float64'])
+shape = simple_fixture('shape', [(64, 64), (128, 128)])
+
+
+def test_operator(dtype, shape):
+    """Test ShearlabOperator operator."""
+    rel = 100 * np.finfo('float32').resolution
+
+    space = odl.uniform_discr([-1, -1], [1, 1], shape, dtype=dtype)
+
+    op = odl.contrib.pyshearlab.ShearlabOperator(space, num_scales=2)
+
+    phantom = odl.phantom.shepp_logan(space, True)
+
+    # Test evaluation
+    y = op(phantom)
+
+    # <Ax, Ax> = <x, AtAx>
+    ax = op.adjoint(y)
+    assert pytest.approx(y.inner(y), rel=rel) == phantom.inner(ax)
```

Comment:
I guess this is my code by these days we try to write this the other way around:

```
assert something == pytest.approx(something_else)
```

since this flows better with english "something is approximately something else"